fix-point/src/CopyFiles.hs

```diff
@@ -0,0 +1,45 @@
+module CopyFiles where
+
+import Control.Applicative ((<$>))
+import Control.Exception (throw)
+import Control.Monad (when,forM_)
+import Data.Functor ((<$>))
```

Comment:
You exported `<$>` operator twice. But you don't even need to export it all, it's available in `Prelude` by default.